docs/Training-on-Amazon-Web-Service.md

```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
 1. In the Unity Editor, load a project containing an ML-Agents environment (you
    can use one of the example environments if you have not created your own).
 2. Open the Build Settings window (menu: File > Build Settings).
-3. Select Linux as the Target Platform, and x86_64 as the target architecture.
-4. Check Headless Mode (If you haven't setup the X Server).
+3. Select Linux as the Target Platform, and x86_64 as the target architecture 
+(the default x86 currently doesn't work).
+4. Check Headless Mode (If you haven't setup the X Server, if you don't use
```

Comment:
"Check Headless Mode (If you haven't setup the X Server, if you don't use Headless Mode, you have to setup the X Server to enable the training)" --> "Check Headless Mode if you have not setup the X Server. (If you do not use Headless Mode, you have to setup the X Server to enable training)"